prow/cluster/resources/monitoring/.gitignore

```diff
@@ -0,0 +1 @@
+charts/*.tgz
```

Comment:
We need to decide on a design pattern here. Either we commit the dependent charts into the repo and use them, or every time we deploy, we need to run `helm dep build` in order to get them.

Reply:
Monitoring will be not installed during prow cluster installation. It will be separate step described in instruction. We don't want to commit charts.